examples/MQ/serialization/processor.cxx

```diff
@@ -1,56 +1,63 @@
 /********************************************************************************
- * Copyright (C) 2014-2022 GSI Helmholtzzentrum fuer Schwerionenforschung GmbH  *
+ * Copyright (C) 2023 GSI Helmholtzzentrum fuer Schwerionenforschung GmbH       *
  *                                                                              *
  *              This software is distributed under the terms of the             *
  *              GNU Lesser General Public Licence (LGPL) version 3,             *
  *                  copied verbatim in the file "LICENSE"                       *
  ********************************************************************************/
 
-#ifndef EX1PROCESSOR_H
-#define EX1PROCESSOR_H
-
-#include "FairMQ.h"   // for fair::mq::Device
+// this example
+#include "ExHeader.h"
 #include "MyDigi.h"
 #include "MyHit.h"
+
+// FairRoot
+#include "BoostSerializer.h"
+#include "FairMQ.h"   // for fair::mq::Device, fair::mq::Parts
+#include "FairRunFairMQDevice.h"
 #include "RootSerializer.h"
 
-#include <TClonesArray.h>
+// ROOT
 #include <TMath.h>
+
+// std
 #include <memory>
 
-class Ex1Processor : public fair::mq::Device
+namespace bpo = boost::program_options;
+
+class Processor : public fair::mq::Device
 {
   public:
```

Comment:
```suggestion
struct Processor : fair::mq::Device
{
```

If you like `class` for consistency, fine with me, if `struct` is not objected by anyone, this could be shorter :P

Reply:
+1 for struct